csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +82,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Outputs a <paramref name="string" /> to the console. The method is only executed if the application was compiled with the DEBUG directive.</para>
+        /// <para>Выводит строку(<paramref name="string" />) в консоль. Метод выполняется только в том случае, если приложение было скомпилировано с директивой DEBUG.</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>The string to be outputted to the console.</para>
+        /// <para>Строка выводимая в консоль.</para>
+        /// </param>
         [Conditional("DEBUG")]
         public static void Debug(string @string) => Console.WriteLine(@string);
 
+        /// <summary>
+        /// <para>Writes the text representation of the specified array of objects(<paramref name="args"/>), followed by the current line terminator, to the standard output stream using the specified format information(<paramref name="format"/>).</para>
+        /// <para>Записывает текстовые представления заданного массива объектов(<paramref name="args"/>), за которым следует текущий признак конца строки, в стандартный выходной поток с использованием заданных сведений о форматировании(<paramref name="format"/>).</para>
```

Comment:
Use direct references to arguments here. Copy the note about `DEBUG` directive here at the and of the comment.